policyengine_canada/variables/gov/provinces/yt/tax/income/credits/childrens_fitness_tax_credit/child_physical_activities_fees.py

```diff
@@ -0,0 +1,8 @@
+from policyengine_canada.model_api import *
+
+
+class child_physical_activities_fees(Variable):
```

Comment:
Since we're filtering to children later, let's make this variable more general and capture fees for any fitness activities for any person, not only children